plugins/org.python.pydev.parser/src/org/python/pydev/parser/grammarcommon/DefaultPythonGrammarActions.java

```diff
@@ -481,9 +481,23 @@ public void makeString(Token t, int quotes, Str strToFill) {
         }
         quoteChar = s.charAt(start);
         if (quoteChar == 'r' || quoteChar == 'R') {
-            //raw string (does not decode slashes)
-            String str = s.substring(quotes + start + 1, s.length() - quotes);
-            //System.out.println("out: "+str);
+            if (start == 0) {
+                char quoteCharAux = s.charAt(1);
+                if (quoteCharAux == 'f' || quoteCharAux == 'F') {
+                    fstring = true;
+                    start++;
+                } else if (quoteCharAux == 'b' || quoteCharAux == 'B') {
+                    bstring = true;
+                    start++;
+                }
+            }
+            String str;
+            if (bstring) {
+                str = 'b' + s.substring(quotes + start, s.length() - quotes + 1);
```

Comment:
Humm, why is this done here? I don't think we need special treatment for the str here...

Reply:
bstrings in Python 3.8 always output with a standard style ( " b'{foo}' " )

Reply:
Python 3.8 output bstring in " b'xxxxx' " style

Reply:
I checked it here and it's not really consistent with others, so, it shouldn't be done.